sequencer.c

```diff
@@ -2032,6 +2039,20 @@ static int pick_commits(struct todo_list *todo_list, struct replay_opts *opts)
 			*end_of_arg = '\0';
 			res = do_exec(item->arg);
 			*end_of_arg = saved;
+
+                        /* Reread the todo file if it has changed. */
+                        if (res == 0) {
```

Comment:
The mailing list reviews are sometimes, how to put it, uhm, let's say, special in their ways. They would most likely take offense at `res == 0`, claiming that the rest of the code uses `!res` for these kinds of tests. I recommend changing it before the reviewers go wild.